src/services/WolfDen.API/Program.cs

```diff
@@ -1,12 +1,14 @@
 using System.Reflection;
+using FluentValidation;
 using Microsoft.EntityFrameworkCore;
 using QuestPDF.Infrastructure;
-using WolfDen.Infrastructure.Data;
-using FluentValidation;
+using sib_api_v3_sdk.Client;
 using WolfDen.Application.Requests.Queries.Attendence.DailyAttendanceReport;
+using WolfDen.Infrastructure.Data;
 
 
 var builder = WebApplication.CreateBuilder(args);
+Configuration.Default.ApiKey.Add("api-key", builder.Configuration["BrevoApi:ApiKey"]);
```

Comment:
Is this required, Can we use optionsmonitor inside the class where api key is used